src/server/scripts/Outland/TempestKeep/Mechanar/boss_mechano_lord_capacitus.cpp

```diff
@@ -134,9 +139,17 @@ class boss_mechano_lord_capacitus : public CreatureScript
                             break;
                         case EVENT_SUMMON_NETHER_CHARGE:
                         {
-                            Position pos = me->GetRandomNearPosition(5.0f);
-                            me->SummonCreature(NPC_NETHER_CHARGE, pos, TEMPSUMMON_TIMED_DESPAWN, 18000);
-                            events.ScheduleEvent(EVENT_SUMMON_NETHER_CHARGE, 10s);
+                            uint32 spellId = 0;
+                            uint32 netherChargeTimer = DUNGEON_MODE(urand(9000, 11000), urand(2000, 5000));
+                            switch (urand(0, 3))
```

Comment:
if I remember right there is a 1 liner that returns a random spell id from a list of spells, I'll check if I can find it

Reply:
RAND(spellid1, spellid2, spellid3, spellid4);

Reply:
containers::SelectRandomContainerElement

Reply:
RAND is more suitable here